activerecord/lib/active_record/errors.rb

```diff
@@ -373,6 +373,14 @@ class ReadOnlyRecord < ActiveRecordError
   end
 
   # Raised on attempt to lazily load records that are marked as strict loading.
+  # This avoids <tt>N+1</tt> queries.
+  #
+  # You can resolve this error by eager loading marked records before accessing
+  # them.  The
+  # {Eager Loading Associations}[https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations]
+  # guide covers solutions, such as using
+  # {ActiveRecord::Base.includes}[rdoc-ref:QueryMethods#includes] or setting
+  # {strict_loading}[rdoc-ref:QueryMethods#strict_loading].
```

Comment:
```suggestion
  # {ActiveRecord::Base.includes}[rdoc-ref:QueryMethods#includes].
```

If someone turned it on it doesn't make sense to document changing the setting IMO.